mllib/src/test/scala/org/apache/spark/ml/clustering/KMeansSuite.scala

```diff
@@ -118,6 +138,11 @@ object KMeansSuite {
     sql.createDataFrame(rdd)
   }
 
+  def generateKMeansModel(dim: Int, k: Int): KMeansModel = {
```

Comment:
`generateRandomKMeansModel`. Added an random seed with default value into this signature to make the whole thing kind of deterministic.